tests/mgbench/benchmark.py

```diff
@@ -235,45 +255,45 @@ def filter_benchmarks(generators, patterns):
                     else:
                         count = count * 10
                 memgraph.stop()
-                config.set_value(*config_key, value={
-                    "count": count,
-                    "duration": args.single_threaded_runtime_sec})
+                config.set_value(*config_key, value={"count": count, "duration": args.single_threaded_runtime_sec})
             else:
-                print("Using cached query count of", cached_count["count"],
-                      "queries for", cached_count["duration"],
-                      "seconds of single-threaded runtime.")
-                count = int(cached_count["count"] *
-                            args.single_threaded_runtime_sec /
-                            cached_count["duration"])
+                print(
+                    "Using cached query count of",
+                    cached_count["count"],
+                    "queries for",
+                    cached_count["duration"],
+                    "seconds of single-threaded runtime.",
+                )
+                count = int(cached_count["count"] * args.single_threaded_runtime_sec / cached_count["duration"])
 
             # Benchmark run.
             print("Sample query:", get_queries(func, 1)[0][0])
-            print("Executing benchmark with", count, "queries that should "
-                  "yield a single-threaded runtime of",
-                  args.single_threaded_runtime_sec, "seconds.")
-            print("Queries are executed using", args.num_workers_for_benchmark,
-                  "concurrent clients.")
+            print(
+                "Executing benchmark with",
+                count,
+                "queries that should yield a single-threaded runtime of",
+                args.single_threaded_runtime_sec,
+                "seconds.",
```

Comment:
is that extra comma is intentional?

Reply:
It's put automatically by the formatter. It gets back when I remove it :(

Reply:
Black always adds a trailing comma when it does not change anything semantically since it makes consequent addition of stuff easier, plus if something is changed only one line will be changed and reduces changes of git conflicts with other files, so I would say this is a good practice to follow with formats that allow this